### PR TITLE
Test commit

### PR DESCRIPTION
**Pull request by suneetha-njta**
Caption Update

### CODE REVIEW

File: _slides/11-suneetha-njta.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 layout: slide
 title: "Welcome to our slide deck!"
 ---
-
-Caption here
+Git Demo
```
Comment (beardofedu) [inline _slides/11-suneetha-njta.md:5]: great comment!

File: _slides/11-suneetha-njta.md [hunk 2]
```diff
@@ -2,8 +2,7 @@
...
 
 ![droidtocat](https://octodex.github.com/images/droidtocat.png)
```
Comment (beardofedu) [inline _slides/11-suneetha-njta.md]: If you want a different image, grab it from here: https://octodex.github.com

File: _slides/11-suneetha-njta.md [hunk 3]
```diff
@@ -2,8 +2,7 @@
 layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _slides/11-suneetha-njta.md]: You should change this to include your username ... 

```suggestion
title: "Welcome to suneetha-njta's slide!"
```

**Review by beardofedu — CHANGES_REQUESTED** [review]
Just fix the `title:` and this is good to 🚢

File: _slides/11-suneetha-njta.md (hunk 2 above)
Comment (beardofedu) [inline _slides/11-suneetha-njta.md]:
```suggestion
![octocat robot](https://octodex.github.com/images/droidtocat.png)
```

Comment (suneetha-njta) [reply _slides/11-suneetha-njta.md]: thanks for suggestion

**Review by beardofedu — APPROVED** [review]

**Pull request merged**